paper-password-input.html

```diff
@@ -100,7 +103,14 @@
 		},
 		_toggleVisibile: function() {
 			this.visible = !this.visible;
-		},
+		},		
+		ready: function() {
+        		super.ready();
```

Comment:
[`super.ready()`](https://www.polymer-project.org/2.0/docs/devguide/registering-elements) is referred to in the docs and is useful if you have validators because you don't want it to work until you have all the input values.

Reply:
Certainly. Looking at this "class" though: The element is still using polymer 1.x constructs, and not ES6 classes -- so there simply is no super to call :)

Reply:
I didn't even think about that!